react-app/src/components/ProposalDetailScreen/ProposalHeader.tsx

```diff
@@ -250,17 +227,14 @@ const ProposalActionArea: React.FC<ProposalActionAreaProps> = (props) => {
         "flex-col",
         "sm:flex-row",
         "sm:justify-between",
-        "sm:justify-center",
-        "mt-4"
+        "sm:justify-center"
       )}
     >
       <div
         className={cn(
           "flex",
           "flex-row",
           "gap-x-3",
-          "sm:mb-0",
-          "mb-3",
```

Comment:
seems need some space between reaction buttons and vote button in mobile view?

<img width="368" alt="image" src="https://user-images.githubusercontent.com/106721852/179934391-e907df22-2a49-4d82-ac60-32cbe7b7acf8.png">

Reply:
Adjusted along with the size of the button as well

<img width="420" alt="Screenshot 2022-07-21 at 12 22 39 PM" src="https://user-images.githubusercontent.com/18374475/180129440-ad8507e6-99f9-4e7f-9bb5-f21a434201a3.png">